src/test/kotlin/org/opensearch/replication/BasicReplicationIT.kt

```diff
@@ -50,47 +50,45 @@ class BasicReplicationIT : MultiClusterRestTestCase() {
+        var response = leader.index(IndexRequest(leaderIndex).id("1").source(source), RequestOptions.DEFAULT)
+        assertThat(response.result).isEqualTo(Result.CREATED)
+
+        assertBusy({
+            val getResponse = follower.get(GetRequest(followerIndex, "1"), RequestOptions.DEFAULT)
+            assertThat(getResponse.isExists).isTrue()
+            assertThat(getResponse.sourceAsMap).isEqualTo(source)
+        }, 60L, TimeUnit.SECONDS)
+
+        // Ensure force merge on leader doesn't impact replication
+        for (i in 2..5) {
+            response = leader.index(IndexRequest(leaderIndex).id("$i").source(source), RequestOptions.DEFAULT)
+            assertThat(response.result).isEqualTo(Result.CREATED)
+        }
+        leader.indices().forcemerge(ForceMergeRequest(leaderIndex), RequestOptions.DEFAULT)
+        for (i in 6..10) {
+            response = leader.index(IndexRequest(leaderIndex).id("$i").source(source), RequestOptions.DEFAULT)
+            assertThat(response.result).isEqualTo(Result.CREATED)
+        }
+        assertBusy({
+            for (i in 2..10) {
+                val getResponse = follower.get(GetRequest(followerIndex, "$i"), RequestOptions.DEFAULT)
                 assertThat(getResponse.isExists).isTrue()
                 assertThat(getResponse.sourceAsMap).isEqualTo(source)
-            }, 60L, TimeUnit.SECONDS)
-
-            // Ensure force merge on leader doesn't impact replication
-            for (i in 2..5) {
-                response = leader.index(IndexRequest(leaderIndex).id("$i").source(source), RequestOptions.DEFAULT)
-                assertThat(response.result).isEqualTo(Result.CREATED)
             }
-            leader.indices().forcemerge(ForceMergeRequest(leaderIndex), RequestOptions.DEFAULT)
-            for (i in 6..10) {
-                response = leader.index(IndexRequest(leaderIndex).id("$i").source(source), RequestOptions.DEFAULT)
-                assertThat(response.result).isEqualTo(Result.CREATED)
-            }
-            assertBusy({
-                for (i in 2..10) {
-                    val getResponse = follower.get(GetRequest(followerIndex, "$i"), RequestOptions.DEFAULT)
-                    assertThat(getResponse.isExists).isTrue()
-                    assertThat(getResponse.sourceAsMap).isEqualTo(source)
-                }
-            }, 60L, TimeUnit.SECONDS)
-
-            // Force merge on follower however isn't allowed due to WRITE block
-            Assertions.assertThatThrownBy {
-                follower.indices().forcemerge(ForceMergeRequest(followerIndex), RequestOptions.DEFAULT)
-            }.isInstanceOf(OpenSearchStatusException::class.java)
-                .hasMessage("OpenSearch exception [type=cluster_block_exception, reason=index [$followerIndex] " +
-                        "blocked by: [FORBIDDEN/1000/index read-only(cross-cluster-replication)];]")
-
-        } finally {
-            follower.stopReplication(followerIndex)
```

Comment:
Lets remove the calls from other integration tests as well - StopIT, StartIT etc?

Reply:
ok